cpp_examples/main.cpp

```diff
@@ -2,6 +2,8 @@
 #include "taichi/ir/statements.h"
 #include "taichi/program/program.h"
 
+namespace taichi {
```

Comment:
if we surround `main()` with `nampespace xx{}`, then who's the program entry function now?